Mobile Buy SDK/Mobile Buy SDK Tests/BUYFakeSafariController.m

```diff
@@ -0,0 +1,60 @@
+//
+//  BUYFakeSafariController.m
+//  Mobile Buy SDK
+//
+//  Created by Shopify.
+//  Copyright (c) 2015 Shopify Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import "BUYFakeSafariController.h"
+
+#pragma mark - Delegator -
+
+@interface BUYFakeDelegator : NSObject
+
+@property (weak, nonatomic) id delegate;
+
+@end
+
+@implementation BUYFakeDelegator
+
+@end
+
+#pragma mark - Fake Controller -
+
+@interface BUYFakeSafariController ()
+
+@end
+
+@implementation BUYFakeSafariController
+
+- (instancetype)initWithURL:(NSURL *)URL {
```

Comment:
The init methods exist on the fake safari controller because they mimic the real safari controller. The "readerMode" was set to `YES` for no reason, just a coin flip between the two values. For consistency, I included both initializers that are available on `SFSafariViewController`.